code/src/Components/Form.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import FirstQuestion from './FirstQuestion';
+import SecondQuestion from './SecondQuestion';
+import ThirdQuestion from './ThirdQuestion';
+import FourthQuestion from './FourthQuestion';
+import FifthQuestion from './FifthQuestion';
```

Comment:
Good structure, but I would ask for more explanatory names on the questions so I know if they are text-input, radio-button etc.